.eslintrc.json

```diff
@@ -0,0 +1,114 @@
+{
+
+    "env": {
+        "browser": true, // // Browser global variables.
```

Comment:
Out of curiosity, why two sets of `//`? I was under the impression that comments are not supported in JSON ([ref](https://stackoverflow.com/questions/244777/can-comments-be-used-in-json)) so it's kinda cool to see that this works.

One alternative to comments in JSON is to convert this .eslintrc.json file to .eslintrc.js

Reply:
No problem!

The eslintrc files that are generated by `yarn eslint --init` actually allow comments in `.json` file types. I thought the same thing until I saw a ton of examples where comments were allowed. I have a `.js` file that I tested as well, and it may be good for future use because you can include functions that offer additional parameters for linting needs. 

Also, I doubled those `//` out of precaution while developing, but I guess they could be removed now. Those comments were mainly present to explain to everyone what each piece of the linter was doing because it had a lot of parts to it.

Reply:
Thanks for explaining!

Totally agreed the comments are helpful and we should preserve them. I'm okay with removing the duplicate `//` but obviously not a priority.